public/docs/ts/latest/guide/architecture.jade

```diff
@@ -408,12 +392,12 @@ block dart-bool
 
 +makeExcerpt('app/hero-detail.component.html', 'ngModel')
 :marked
-  Angular ships with a small number of other directives that either alter the layout structure
+  Angular includes a small number of other directives that either alter the layout structure
```

Comment:
includes -> contains

"includes" (to me) is a synonym for "imports", so let's not use "includes" unless its meaning is very clear.